extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java

```diff
@@ -169,7 +170,7 @@ private void addService(BuildProducer<ServiceProviderBuildItem> services,
                     service);
             services.produce(new ServiceProviderBuildItem(serviceClass.getName(), implementations.toArray(new String[0])));
 
-            reflective.produce(new ReflectiveClassBuildItem(true, true, false, implementations.toArray(new String[0])));
+            reflective.produce(new ReflectiveClassBuildItem(true, true, true, implementations.toArray(new String[0])));
```

Comment:
Maybe it makes sense to only have `Precondition` do this and keep the others like they are?

Reply:
We can do that. I thought so too but I think the reason I went with this implementation is that the others do not have fields that are set reflectively.

Reply:
We can make it explicit and it won't be a harm.

Reply:
Done